src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -384,7 +384,7 @@
             Ref("SamplingExpressionSegment"),
             min_times=1,
         ),
-        Ref("TableAliasExpressionSegment", optional=True),
+        Ref("AliasExpressionSegment", optional=True),
```

Comment:
Can we make this same change in BigQuery?

Reply:
can do!

Reply:
done!